e2e_playwright/main_menu_test.py

```diff
@@ -22,3 +22,66 @@ def test_main_menu_images(themed_app: Page, assert_snapshot: ImageCompareFunctio
 
     element = themed_app.get_by_test_id("stMainMenuPopover")
     assert_snapshot(element, name="main_menu")
+
+
+def test_renders_settings_dialog_properly(
```

Comment:
There is also `Edit active theme` dialog (Click on `Edit active theme` within the theme dialog) which I think isn't covered yet anywhere. So might make sense to include that one as well in this test .

Reply:
Thanks for the comments. Will fix this!